.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       - run: python -m pip install 'tox<4'
       - run: tox -e lint
   Tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-32-cores
```

Comment:
The same 32-core machines that we're using for LMS. Now that the tests are running in parallel this should make CI much faster.